apollo-client/src/main/java/com/ctrip/framework/apollo/internals/ConfigServiceLocator.java

```diff
@@ -161,7 +161,13 @@ private String getDeprecatedCustomizedConfigService() {
    */
   public List<ServiceDTO> getConfigServices() {
     if (m_configServices.get().isEmpty()) {
-      updateConfigServices();
+      m_executorService.submit(() -> this.tryUpdateConfigServices());
```

Comment:
The default `scheduleAtFixedRate` is too late(5 minutes), 
I think we need to speed up it by submit immediately.

But that will make multiple http request.

There are some solutions:
* Solution 1: Add a rate limiter here
* Solution 2: scheduleAtFixedRate another method `recover` 5 seconds period, use a boolean to mark execute or not, and set the boolean to true in this line.
* Solution 3: Use a state machine to indicate task execute or not, if in executing, then will not submit task.
 
Or use another solution?

Reply:
A rate limiter should be effective.

Reply:
Done, use default 1 qps.

add 3 config

```
apollo.discoveryQPS
apollo.discoveryConnectTimeout
apollo.discoveryReadTimeout
```

let user can custom behavior of discovery